security/voters.rst

```diff
@@ -323,3 +323,45 @@ security configuration:
                 'allow_if_all_abstain' => false,
             ],
         ]);
+
+If none of the built-in strategies fits your use case, define the ``service``
+option to use a custom service as the Access Decision Manager:
```

Comment:
I think we should mention somewhere which interface has to be implemented by this service? (`Symfony\Component\Security\Core\Authorization\AccessDecisionManagerInterface`)